### ISSUE TITLE
[Technical Question] is azure-iot-device-http https or http? What protocol should I use?

### ISSUE THREAD

**Issue by cell2749** [Azure/azure-iot-sdk-node#1015]
Sorry for this might be two questions in one.

(Topic) 
1. If it supports both protocols, how to make sure I am using https? 
I am also using `Message` and `Client` from `aws-iot-device`

2. Scenario: Simple publish of a message(s). (Forwarding messages from device(s) in cloud from one service to another service). There is no need to forward messages from another service to device(s).

Is amqp more suitable for the above scenario?

**Comment by anthonyvercolano** [comment]
Among the protocols Azure IoT supports https.  By chosing the http protocol you are getting https.

As to the rest I am not clear about what you are asking.

**Comment by cell2749** [comment]
> Among the protocols Azure IoT supports https. By chosing the http protocol you are getting https.
> 
> As to the rest I am not clear about what you are asking.

Yes, the main question here is whether the npm module "azure-iot-device-http" is actually https? I had a brief look at the code and it seems that in some scenarios the connection might be http and thus was wondering if that is the case with the mentioned module or not?

**Comment by YoDaMa** [comment]
for all scenarios the connection would be using http over TLS.
